sources/packages/backend/apps/db-migrations/src/sql/Notifications/Add-metadata-column.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE
+  sims.notifications
+ADD
+  COLUMN metadata jsonb;
+
+COMMENT ON COLUMN sims.notifications.metadata IS 'Metadata information on notifications sent such as tracking the various relations associated with the sent notification.';
```

Comment:
I would prefer to not indicate the relationships in the comment as it can mislead someone to believe that the intention is so save DB relationships only here.
Suggestion:
```
Metadata information for a notification to allow a more granular identification or identify further the reason why it was generated.
```